lithium/lithium.py

```diff
@@ -268,7 +268,7 @@ def readTestcaseLine(line):
 def writeTestcase(filename):
     with open(filename, "w") as file:
         file.write(before)
-        for i in range(len(parts)):
+        for i in xrange(len(parts)):
```

Comment:
`xrange/range(len())` is a bad pattern either way. A better one is:

    for part in parts:
        file.write(part)

but in this case `file.writelines(parts)` would be better.